### PR TITLE
Feature/roe 1601 add dual nationality

### PR DESCRIPTION
**Pull request by mouhajer-ch**
### JIRA link

Resolves [ROE-1601](https://companieshouse.atlassian.net/browse/ROE-1601)

### Change description

Add dual nationality

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### CODE REVIEW

File: src/validation/custom.validation.ts [hunk 1]
```diff
@@ -85,6 +85,13 @@ export const checkOptionalDate = (dayStr: string = "", monthStr: string = "", ye
   return true;
 };
 
+export const checkSecondNationality = (nationality, secondNationality) => {
```
Comment (markpit) [inline src/validation/custom.validation.ts:88]: i would add the  "string" types to the parameters

File: src/validation/error.messages.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export enum ErrorMessages {
   ROLE_AND_RESPONSIBILITIES_CORPORATE = "Enter a description of the corporate managing officer’s role and responsibilities",
   ROLE_AND_RESPONSIBILITIES_INDIVIDUAL = "Enter a description of the individual managing officer’s role and responsibilities",
   NATIONALITY = "Enter the individual person’s nationality",
+  SECOND_NATIONALITY = "Second nationality should not be the same as the nationality given",
```
Comment (markpit) [inline src/validation/error.messages.ts:15]: i'd call the message SECOND_NATIONALITY_IS_SAME

File: test/controllers/managing.officer.controller.spec.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ jest.mock('../../src/utils/save.and.continue');
 jest.mock("../../src/utils/logger");
 jest.mock('../../src/middleware/navigation/has.beneficial.owners.statement.middleware');
 
-import { describe, expect, test, jest } from '@jest/globals';
+import { describe, expect, test, jest, beforeEach } from '@jest/globals';
```
Comment (markpit) [inline test/controllers/managing.officer.controller.spec.ts:10]: is the beforeEach needed? i couldn't see it used in the changes

Comment (mouhajer-ch) [reply test/controllers/managing.officer.controller.spec.ts:10]: It is needed on line 89, it was missing, I just added to fix ts error on my IDE

**Review by markpit — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Pull request merged**